jest/jest-tests.ts

```diff
@@ -199,6 +199,16 @@ describe('Assymetric matchers', function () {
         }));
 
         expect.assertions(4);
+
+        interface Test {
+            a: number;
+            b: string;
+        }
+
+        const test: Test = {
```

Comment:
I think it might be helpful to have that an assertion like that here in the test code so in the future people will know why they shouldn't break this use case. It's common for people to try to 'fix' `any`s they come across in type definitions.